### DISCUSSION TITLE
How to install this in ExtJS app.

### DISCUSSION THREAD

**Discussion by maddy65** [apache/skywalking#8550] [Q&A]
I have ExtJS app and I am trying to install this NPM. Can anybody help me how to use this.

 In reactJS app after installing thins comes and sit in package json so I can use smoothly. 

How can we make compatible to sencha.

Will appreciate for Any leads

**Comment by Fine0830** [comment]
What do you want to install in ExtJS app?

**Reply by wu-sheng** [reply]
Add a context, SkyWalking has an NodeJS agent for nodejs backend, and a client-js agent for the browser. Which one are you asking about?

**Reply by maddy65** [reply]
@wu-sheng  I am using extJS for my frontend development, which is getting install through sencha. I wann use Skywalking features, How to use that in extJs .

**Reply by wu-sheng** [reply]
Frontend is separated from server side and browser side. I don't have experience of your technology stack, so ask generally.

**Comment by maddy65** [comment]
we are planning to do browser monitoring using through skwalking client js. As per the document we need to do the install by this command . `npm install skywalking-client-js --save` 

In our application we are using extJS https://www.sencha.com/products/extjs/

How can we use skywalking client js in our application, we don't have npm modules in our application. Is there any alternate where we can download compiled js file and I can use in my application through library.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking-client-js#quick-start

You need to put this code in your browser application, no matter what framework you are using.
